packages/pieces/community/json/src/lib/actions/convert-text-to-json.ts

```diff
@@ -0,0 +1,21 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+
+export const convertTextToJson = createAction({
+  // auth: check https://www.activepieces.com/docs/developers/piece-reference/authentication,
+  name: 'convertTextToJson',
+  displayName: 'convert text to json',
```

Comment:
Nit:  Please change it to `Convert Text to Json` and set the description to empty